### DISCUSSION TITLE
Compose New Email - HTTP 500 Error

### DISCUSSION THREAD

**Discussion by toofast1** [modoboa/modoboa#2241] [Help]
Hello,

When trying to compose an email, after pressing the Send button, I get an HTTP 500 error. This is the response:

`XHRPOST https://mail.domain.com/webmail/?action=compose
[HTTP/1.1 500 Internal Server Error 1068ms]
TypeError at /webmail/
Object of type bytes is not JSON serializable
Request Method: POST
Request URL: https://mail.domain.com/webmail/?action=compose
`

I need to mention that the email doesn't contain any attachments, just a simple test email send to a Gmail address.

Any ideas what could be wrong?

**Comment by tonioo** [comment]
@toofast1 Can you set DEBUG to True in your settings.py, reload uwsgi and try again with you debugger opened? Look at the Network tab, click on the failing request and open the Response tab, you'll have more information.

**Reply by H-A-L-I-S** [reply]
It says: 
`# SECURITY WARNING: don't run with debug turned on in production!`
Is it okay to still activate it?

**Reply by tonioo** [reply]
For debug purposes only
